src/examples/app.js

```diff
@@ -35,7 +35,9 @@ class App extends React.Component {
   render() {
     return (
       <Router>
-        <div style={{ height: '100%' }}>
+        <div
+          style={{ height: '100%', display: 'flex', flexDirection: 'column' }}
```

Comment:
Instead of inline-style, we need to use dedicated fils of.scss to reduce conflict later, what's your opinion on this  @IgnacioFDM